src/style.css

```diff
@@ -0,0 +1,91 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+body * {
+  box-sizing: border-box;
+}
+
+/* Visibility hidden snippet */
+.form .input.focus,
+.form .input.default:focus,
+.form:focus-within .input.default {
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  position: absolute;
+  border: 0;
+  overflow: hidden;
+}
+
+/* Special styles for focus class inputs when their active */
+.form .input.default:focus + .focus,
+.form .input.focus:focus {
+  border-radius: 3px;
+  box-shadow: none;
+}
+
+/* General styles for inputs */
+.form .input,
+.form .input.default:focus + .focus,
+.form .input.focus:focus {
+  position: static;
+  width: 100%;
+  margin-bottom: 40px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-image: url("images/Search.svg");
```

Comment:
you can use the image as background as well, but you don't need another input for this, clean up your code